core/templates/domain/exploration/ExplorationObjectFactorySpec.ts

```diff
@@ -379,4 +384,96 @@ describe('Exploration object factory', () => {
     expect(exploration.getAuthorRecommendedExpIds('second state'))
       .toEqual([]);
   });
+
+  describe('getWrittenTranslationLanguageCodesReadyForDisplay', () => {
+    beforeEach(() => {
+      spyOn(
+        exploration.states,
+        'getAllWrittenTranslationLanguageCodes'
+      ).and.returnValue(['fr']);
+    });
+
+    it('should return no language codes for an exploration that contains' +
```

Comment:
Add a TODO here to remove this test once the other PR is merged (have it point to the same TODO as above).